### PR TITLE
Fast fail GitHub and make xargs work on Linux

### PR DESCRIPTION
**Pull request by torbjornvatn**

### CODE REVIEW

File: brokkr/brokkr.mk [hunk 1]
```diff
@@ -23,7 +23,7 @@ $(_BROKKR_PLUGINS_SENTINEL):
 $(_BROKKR_PLUGINS_MK): .SHELLFLAGS := -c
 $(_BROKKR_PLUGINS_MK): $(_BROKKR_PLUGINS_SENTINEL)
 	@# Clean working dir, except sentinel
-	@find $(_BROKKR_PLUGINS_DIR) -type f -not -name "`basename $(_BROKKR_PLUGINS_SENTINEL)`" | xargs rm
+	@find $(_BROKKR_PLUGINS_DIR) -type f -not -name "`basename $(_BROKKR_PLUGINS_SENTINEL)`" | xargs -r rm
```
Comment (judoole) [inline brokkr/brokkr.mk:26]: Should the `-r` be at the end, like `xargs rm -r`?

Comment (torbjornvatn) [reply brokkr/brokkr.mk:26]: I think so, will fix

Comment (torbjornvatn) [reply brokkr/brokkr.mk:26]: Actually not https://stackoverflow.com/a/36618260

Comment (torbjornvatn) [reply brokkr/brokkr.mk:26]: Because it's an option to xargs not rm

Comment (judoole) [reply brokkr/brokkr.mk:26]: Hmmm...
```
mkdir -p /tmp/jass && echo /tmp/jass | xargs -r rm
rm: /tmp/jass: is a directory
```

Comment (judoole) [reply brokkr/brokkr.mk:26]: While `mkdir -p /tmp/jass && echo /tmp/jass | xargs rm -r` worked

Comment (judoole) [reply brokkr/brokkr.mk:26]: I misunderstood. You are only deleting files anyways.

Comment (torbjornvatn) [reply brokkr/brokkr.mk:26]: Ok, maybe we need it to be `xargs -r rm -r` then

**Review by judoole — APPROVED** [review]

File: brokkr/brokkr.mk (hunk 1 above)
Comment (torbjornvatn) [reply brokkr/brokkr.mk:26]: Yes, I guess that's what the –type f in the find is for

**Pull request merged**